spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSettings.java

```diff
@@ -326,6 +328,126 @@ public void setMinResponseSize(String minResponseSize) {
     @Nullable
     private Compression compression;
 
+    /**
+     * The number of threads for {@link EventLoopGroup} that the {@link Server} uses.
+     */
+    @Nullable
+    private Integer workerGroup;
+
+    /**
+     * The number of threads dedicated to the execution of blocking tasks or invocations.
+     */
+    @Nullable
+    private Integer blockingTaskExecutor;
+
+    /**
+     * The maximum allowed number of open connections.
+     */
+    @Nullable
+    private Integer maxNumConnections;
+
+    /**
+     * The idle timeout of a connection for keep-alive.
+     */
+    @Nullable
+    private Duration idleTimeout;
+
+    /**
+     * The interval of the HTTP/2 PING frame.
+     */
+    @Nullable
+    private Duration pingInterval;
+
+    /**
+     * The maximum allowed age of a connection for keep-alive.
+     */
+    @Nullable
+    private Duration maxConnectionAge;
+
+    /**
+     * The maximum allowed number of requests that can be served through one connection.
+     */
+    @Nullable
+    private Integer maxNumRequestsPerConnection;
+
+    /**
+     * The initial connection-level HTTP/2 flow control window size.
+     */
+    @Nullable
+    private String http2InitialConnectionWindowSize;
+
+    /**
+     * The initial stream-level HTTP/2 flow control window size.
+     */
+    @Nullable
+    private String http2InitialStreamWindowSize;
+
+    /**
+     * The maximum number of concurrent streams per HTTP/2 connection.
+     */
+    @Nullable
+    private Long http2MaxStreamsPerConnection;
+
+    /**
+     * The maximum size of HTTP/2 frame that can be received.
+     */
+    @Nullable
+    private String http2MaxFrameSize;
+
+    /**
+     * The maximum size of headers that can be received.
+     */
+    @Nullable
+    private String http2MaxHeaderListSize;
+
+    /**
+     * The maximum length of an HTTP/1 response initial line.
+     */
+    @Nullable
+    private String http1MaxInitialLineLength;
+
+    /**
+     * The maximum length of all headers in an HTTP/1 response.
+     */
+    @Nullable
+    private String http1MaxHeaderSize;
+
+    /**
+     * The maximum length of each chunk in an HTTP/1 response content.
+     */
+    @Nullable
+    private String http1MaxChunkSize;
+
+    /**
+     * The format of the {@link Server}'s access log.
+     */
+    @Nullable
+    private String accessLogFormat;
```

Comment:
How about adding AccessLog type as we do in dropwizard so that a user can specify pre-defined type?
```
access-log:
  type: common
// or
access-log:
  type: custom
  format: "Your own log format"
```
https://github.com/line/armeria/blob/master/dropwizard2/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaSettings.java#L930